src/modules/map/components/olMap/olMapUtils.js

```diff
@@ -41,9 +41,16 @@ export const registerLongPressListener = (map, callback, delay = 300) => {
 	});
 };
 
+
+/**
+ * Checks whether a layer contains features or not.
+ * @param {Layer} layer
+ * @returns {boolean}
+ */
 export const isEmptyLayer = (layer) => {
```

Comment:
We should rename this function to a more precise name e.g. `hasFeatures`  (OL knows much more layer types than the Vector Layer. What does _empty_ mean in that context?).

I wonder why it should return `true`  for the case of an undefined layer 🤓